threshold: 0.3
possible_classes:
    people: [Person]
    food: [master_chef_can, cracker_box, sugar_box, tomato_soup_can, mustard_bottle, tuna_fish_can, pudding_box, gelatin_box, potted_meat_can, banana, strawberry, apple, lemon, peach, pear, orange, plum]
    kitchen: [Pitcher base, Srub cleanser bottle, Windex Spray bottle, Bowl, Mug, Scotch brite dobie sponge, Plate, Fork, Spoon, Spatula]
    tools: [Large marker, Clamps]
    shape: [Mini soccer ball, Soft ball, Baseball, Tennis ball, Racquetball, Golf ball, Chain, Foam brick, Dice, Marbles, Cups]
    task: [Colored wood blocks, 9-peg-hole test, Toy airplane, Lego duplo, Rubikâ€™s cube]
all_classes:
    #Note: fill this list with the class list from yolov5 dataset.yaml file used to train the model, or attempt to get this list by printing the state dictionary in the model file. The class names should be in the same order.
    []
model_file: larc2021_go_and_get_it.pt
